lib/capistrano/lazy_cleanup/defaults.rb

```diff
@@ -0,0 +1,14 @@
+require 'rake'
+
+module Capistrano
+  module LazyCleanup
+    module Defaults
+    end
+  end
+end
+
+# TODO: If Capistrano::Plugin system becomes stable, refactor these code.
+
+Rake::Task.define_task('load:defaults') do
+  set_if_empty(:lazy_cleanup_old_releases_path_template, "#{fetch(:tmp_dir, '/tmp')}/capistrano-lazy_cleanup_old_releases.XXXXXXXXXX")
```

Comment:
Can we shorten the `capistrano-lazy_cleanup_old_releases` to `cap-lazy_cleanup`?
It is still readable to developers who uses `cap` as their command for deployments.

IMHO, adding `fetch(:application)` in the path can further help developers.